ynr/apps/parties/serializers.py

```diff
@@ -47,4 +47,4 @@ class Meta:
 class MinimalPartySerializer(PartySerializer):
     class Meta:
         model = Party
-        fields = ("ec_id", "name")
+        fields = ("ec_id", "name", "legacy_slug")
```

Comment:
This makes the change in the serializer used by `/next`, but WhoCIVF is still calling `/v0.9`. Don't we want to change it there too?

Reply:
Oh sorry, I took the above comment as an approval and merged…

I'm working on a PR now to update WCIVF to use the `next` API…